s3cloudfront/s3-cloudfront.tf

```diff
@@ -0,0 +1,141 @@
+terraform {
+  backend "s3" {
```

Comment:
I don't believe that we can interpolate variables into the backend configuration, which is why the bash script in the `infra` folder is dynamically creating the backend configuration prior to running Terraform: https://github.com/FightPandemics/infrastructure/blob/master/infra/deploy.sh#L38-L48